e2e/utils.ts

```diff
@@ -83,12 +84,26 @@ export function runCLI(
   }
 }
 
-export function newApp(name: string): string {
-  return runCLI(`generate app ${name}`);
+export function newApp(name: string, schematics?: string): string {
```

Comment:
None of these helpers functions should differ for bazel and non-bazel setups. The user should never worry about passing the right collection, and instead should just call:

```
ng g app myapp
ng g component mycomponent
```

To make it work, update `.angular-cli.json` to set the default collection to `@nrwl/bazel`.